packages/playground/src/layouts/DefaultHeaderLayout.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactNode } from 'react'
+
+interface DefaultHeaderProps {
+  children: ReactNode
```

Comment:
no need to define `children` as a prop in the interface. `React.FC` already had that.